systemtest/src/test/java/io/strimzi/systemtest/AbstractST.java

```diff
@@ -421,12 +421,10 @@ protected void verifyLabelsOnPods(String clusterName, String podType, String app
 
     void verifyLabelsForCRDs() {
         LOGGER.info("Verifying labels for CRDs");
-        kubeClient().listCustomResourceDefinition().stream()
-            .filter(crd -> crd.getMetadata().getName().startsWith("kafka"))
-            .forEach(crd -> {
-                LOGGER.info("Verifying labels for custom resource {}", crd.getMetadata().getName());
-                assertThat(crd.getMetadata().getLabels().get("app"), is("strimzi"));
-            });
+        String crds = cmdKubeClient().exec("get", "crds", "--selector=app=strimzi", "-o", "jsonpath='{.items[*].metadata.name}'").out();
```

Comment:
Question: Why do we use kubectl commands over using the fabric8 api?

Reply:
As I mentioned in description, the only problem here was that f8 returns this exception: https://gist.github.com/im-konge/2c184783f0f3daebb4e5e4e89b6d5b07

Reply:
So after @stanlyDoge upgrade the fabric8 to new version, I hope it will be fixed and this will be changed back

Reply:
Absolutely, sorry missed the description :P